tests/test_crawler.py

```diff
@@ -245,3 +256,10 @@ def test_crawler_runner_bootstrap_failed_for_several(self):
         yield runner.crawl(NoRequestsSpider)
 
         self.assertEqual(runner.bootstrap_failed, True)
+
+    @defer.inlineCallbacks
+    def test_asyncio_supported(self):
```

Comment:
This really just checks the current reactor via `is_asyncio_supported()`, as `CrawlerRunner` doesn't do anything special here and doesn't run any new processes and the reactor is global, so maybe this test can be dropped.